apps/climatemappedafrica/src/payload/globals/HURUMap/Profile.js

```diff
@@ -1,6 +1,93 @@
+import HURUMapURL from "../../fields/HURUMapURL";
+import LocationSelect from "../../fields/LocationSelect";
+import ProfileSelect from "../../fields/ProfileSelect";
+
 const Profile = {
   label: "Profile",
   fields: [
+    {
+      name: "hurumapAPIURL",
+      label: "HURUMap API BASE URL",
```

Comment:
```suggestion
      name: "url",
      label: "HURUmap NG URL",
```
We already know it's HURUmap Profile, there is no need to repeat it at field level.